metadata-ingestion/src/datahub/ingestion/source/aws/glue.py

```diff
@@ -163,6 +163,11 @@ class GlueSourceConfig(
     stateful_ingestion: Optional[StatefulStaleMetadataRemovalConfig] = Field(
         default=None, description=""
     )
+    extract_delta_schema_from_parameters: Optional[bool] = Field(
+        default=False,
+        description="If enabled, delta schemas can be alternatively fetched from table parameters "
+        "(https://github.com/delta-io/delta/pull/2310)",
```

Comment:
```suggestion
    extract_delta_schema_from_parameters: Optional[bool] = Field(
        default=False,
        description="If enabled, delta schemas can be alternatively fetched from table parameters."
```

The PR is already present in code comment later so need not be repeated.